plugins/cpp/model/include/model/cpptype.h

```diff
@@ -51,6 +51,11 @@ struct CppType : CppEntity
   bool isAbstract = false;
   bool isPOD = false;
 
+  // Size and alignment is expressed in C++ standard bytes (sizeof(char)).
+  // Default value 0 means it couldn't have been obtained from Clang.
+  std::size_t size;
+  std::size_t alignment;
```

Comment:
In the comment you are writing about 0 as a default value, however these members are not initialized.

Reply:
An alignment of `x` bytes means that instances of the type should have their first byte at a memory address that is divisible by `x`. It is to ensure that member data accesses are optimal. See http://en.cppreference.com/w/c/language/object

Reply:
Is there any advantage of storing this information regarding code comprehension? Is there any reasonable use-case why somebody may be interested in the alignment? I'm not sure if any use-case would be worth for this storage.

Reply:
(This should have went as reply to the other comment but the phone client is behaving strange.) It was @zporky's idea to store and present these two "memory model related" labels for the types. For alignment, his example was custom allocators.

Reply:
I still don't think it would be worth extending database size with storing this info, not to mention that considering "reparse" as a future feature this will be reproducible. It is not really related to code comprehension.